helm/monitoring/loki-values.yaml

```
@@ -0,0 +1,17 @@
+    accessModes:
+      - ReadWriteOnce
+    size: 10Gi                    # 초기에는 10Gi 정도로 시작 가능 (운영에 따라 조정)
+
+promtail:
+  enabled: true
+  config:
+    clients:
+      - url: http://loki.monitoring.svc:3100/loki/api/v1/push
+
+grafana:
+  enabled: false
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)